src/core_ocean/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -713,7 +709,7 @@ subroutine ocn_vmix_coefs_cvmix_build(meshPool, statePool, forcingPool, diagnost
             ! add convective mixing to vertical viscosity/diffusivity
             ! if using KPP, then do not apply convective mixing within the ocean boundary layer
             if(config_use_cvmix_kpp) then
-               do k = int(indexBoundaryLayerDepth(iCell)) + 1, maxLevelCell(iCell)
+               do k = int(ceiling(indexBoundaryLayerDepth(iCell))) + 1, maxLevelCell(iCell)
```

Comment:
`ceiling` returns an integer by default ([GCC documentation](https://gcc.gnu.org/onlinedocs/gfortran/CEILING.html), [Intel documentation](https://software.intel.com/content/www/us/en/develop/documentation/fortran-compiler-developer-guide-and-reference/top/language-reference/a-to-z-reference/c-to-d/ceiling.html)), so `int(ceiling(` is redundant.

Reply:
Thanks, fixed.